application/src/messenger.cpp

```diff
@@ -53,6 +63,9 @@ void Messenger::handleResponse(QNetworkReply *reply)
             case Requester::SIGN_UP:
                 handleSignupResponse(reply);
                 break;
+            case Requester::SIGN_MESSAGE:
+                qDebug()<<"Сервер сохранил";
```

Comment:
не пиши лог на русском языке
```suggestion
                qDebug() << "Сервер сохранил";
```